squash_wip_test.go

```diff
@@ -297,3 +346,31 @@ squash c51a56d manual commit
 	result, _ := ioutil.ReadFile(input)
 	equals(t, expected, string(result))
 }
+
+func wipCommit(t *testing.T, configuration Configuration, filename string) {
+	start(configuration)
+	createFile(t, filename, "irrelevant")
```

Comment:
could you use `"contentIrrelevant"` in all cases? Little bit easier to read in github PRs.

Reply:
I agree